services/issue/pull.go

```diff
@@ -51,14 +51,14 @@ func PullRequestCodeOwnersReview(ctx context.Context, issue *issues_model.Issue,
 		return nil, err
 	}
 
-	if pr.HeadRepo.IsFork {
-		return nil, nil
-	}
-
 	if err := pr.LoadBaseRepo(ctx); err != nil {
 		return nil, err
 	}
 
+	if pr.BaseRepo.IsFork {
+		return nil, nil
+	}
```

Comment:
That sounds equally wrong:
If we have base repo `A`, fork `B`, and fork `C` of `B`, then PRs from `C` to `B` don't notify code reviewers?
I'd expect one of two things here:
Either we say that the fork does not touch the codeowners file in which case we don't want to trigger notifications, or it modifies it to its likings in which case we do want notifications.
Sounds a bit like the only way to fix this is by adding a setting that is only available for forks, namely whether CODEOWNERS should be honored.
The default value should probably be `false`

Reply:
For now, it probably suffices if we add this comment ↑ as a TODO above these lines

Reply:
Because codowner should only for base repository's pull requests. So yes, if you send pull requests from C -> B or from B -> B, the code owners will not triggered.

The files will always be read from base repository.

Reply:
I don't think we need to do anything more for forked repositories except ensure that CODEOWNERS doesn't take effect in them.

If users fork a repo to submit PRs to upstream, they don't care about CODEOWNERS in the fork.

If users fork a repo to start a new fire with new CODEOWNERS, I don't think they will really fork it, they will clone, rename, and create a new repo.